main.go

```diff
@@ -66,14 +67,15 @@ func loadConfig() kubernetes.Interface {
 
 	// leverages a file|(ConfigMap)
 	// to be located at /etc/eventrouter/config
-	viper.SetConfigType("json")
+	viper.SetConfigType("yaml")
```

Comment:
This is subjective. Makes more sense for this to be configurable than try to enforce our opinion.